### PR TITLE
2.3.2 release

### PR DESCRIPTION
**Pull request by daschuer**
Is there anything left, that holds up the release? 

I think it is worth to wait for: https://github.com/mixxxdj/mixxx/pull/4619

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Mixxx version 2.3, Digital DJ'ing software.
-Copyright (C) 2001-2021 Mixxx Development Team
+Copyright (C) 2001-2022 Mixxx Development Team
```
Comment (ronso0) [inline LICENSE:2]: meeeh, #4596 should have been merged into 2.3
Can you please try to cherry-pick cb47295f20007fb7f89c5cf97ce995e27244cd7f?

Comment (ronso0) [reply LICENSE:2]: I wouldn't have remembered to update this here, so Thanks!

**Review by ronso0 — CHANGES_REQUESTED** [review]
2021 -> 2022

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,9 @@
 * Preserve file creation time when writing metadata on Windows [#4586](https://github.com/mixxxdj/mixxx/pull/4586) [lp1955314](https://bugs.launchpad.net/mixxx/+bug/1955314)
 * Fix handling of file extension when importing and exporting sampler settings [#4539](https://github.com/mixxxdj/mixxx/pull/4539)
 * Fix crash when using an empty directory as resource path using the `--resource-path` command line option [#4575](https://github.com/mixxxdj/mixxx/pull/4575) [lp1934560](https://bugs.launchpad.net/mixxx/+bug/1934560)
+* Added mapping for Pioneer DDJ SB3 controller [#3821](https://github.com/mixxxdj/mixxx/pull/3821)
```
Comment (Holzhaus) [inline CHANGELOG.md:11]:
```suggestion
* Pioneer DDJ-SB3: Add controller mapping [#3821](https://github.com/mixxxdj/mixxx/pull/3821)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,9 @@
...
+* Don't wipe sound config during startup if configured devices are unavailable [#4544](https://github.com/mixxxdj/mixxx/pull/4544)
+* Append selected file extension when exporting to plalist files [#4531](https://github.com/mixxxdj/mixxx/pull/4531) [lp1889352](https://bugs.launchpad.net/mixxx/+bug/1889352)
```
Comment (Holzhaus) [inline CHANGELOG.md:13]:
```suggestion
* Append selected file extension when exporting to playlist files [#4531](https://github.com/mixxxdj/mixxx/pull/4531) [lp1889352](https://bugs.launchpad.net/mixxx/+bug/1889352)
```

Comment (ronso0) [reply CHANGELOG.md:13]: fixed, thanks

File: CHANGELOG.md (hunk 2 above)
Comment (ronso0) [reply CHANGELOG.md:11]: Good idea, let's use this formatting from now on, that makes it much easier to spot mappings in the changelog.

**Review by ronso0 — APPROVED** [review]
Thank you!

**Comment by ywwg** [comment]
> I think it is worth to wait for: #4619

If the contributor isn't able to fix the rebase issues, we could do it for them and preserve the commit history so they still get credit for the work.

**Comment by uklotzde** [comment]
Is it really worth waiting for #4619? It handles an unexpected edge case that never ever affected anyone throughout the years.

We didn't receive a response from the first-time contributor (on-boarding still required!) since days.

**Review by uklotzde — APPROVED** [review]
Merge?

**Comment by Holzhaus** [comment]
I'm wondering why this PR is even blocked by that PR? Is this just to avoid an additional changelog PR? As long as we don't actually tag 2.3.2, there should be no issue with merging this now, right?

**Comment by daschuer** [comment]
It is just a matter of limiting the number of commits where the tag version does not match the CMake version. 
I think I will find time to take over the rebasing in  #4619 tonight.

**Review by ywwg — APPROVED** [review]
done in #4635! thank you

**Comment by uklotzde** [comment]
Add a final changelog entry for #4635?

**Comment by daschuer** [comment]
Done.

**Pull request merged**